tests/js/errorMiddleware/errorMiddleware-tests.js

```diff
@@ -0,0 +1,149 @@
+/*
+
+    Tests for the "error handling" middleware.
+
+*/
+"use strict";
+var fluid = require("infusion");
+var gpii  = fluid.registerNamespace("gpii");
+
+// Load all of the components to be tested and our test cases
+require("../includes.js");
+
+require("./errorMiddleware-caseholder");
+
+fluid.registerNamespace("gpii.tests.express.errorMiddleware.errorPitcher");
+gpii.tests.express.errorMiddleware.errorPitcher.haltAndCatchFire = function (that, request, response, next) {
+    next({ isError: true, message: "Pray, Mr. Babbage, if you put into the machine wrong figures, will the right answers come out?"});
+};
+
+fluid.defaults("gpii.tests.express.errorMiddleware.errorPitcher", {
+    gradeNames: ["gpii.express.middleware"],
+    invokers: {
+        middleware: {
+            funcName: "gpii.tests.express.errorMiddleware.errorPitcher.haltAndCatchFire",
+            args:     ["{that}", "{arguments}.0", "{arguments}.1", "{arguments}.2"] // request, response, next
+        }
+    }
+});
+
+fluid.defaults("gpii.tests.express.errorMiddleware.testEnvironment", {
+    gradeNames: ["gpii.test.express.testEnvironment"],
+    port:   7551,
+    components: {
+        express: {
+            options: {
+                components: {
+                    simpleErrorPitcher: {
+                        type: "gpii.tests.express.errorMiddleware.errorPitcher",
+                        options: {
+                            namespace: "simpleErrorPitcher",
+                            priority: "first",
+                            path: "/simple",
+                            components: {
```

Comment:
Thanks, I obviously lost the ability to see the detritus leftover from the previous code.   It's gone now.